lib/compat/wordpress-6.2/script-loader.php

```diff
@@ -90,16 +90,18 @@ function gutenberg_resolve_assets_override() {
 	$block_registry = WP_Block_Type_Registry::get_instance();
 
 	foreach ( $block_registry->get_all_registered() as $block_type ) {
-		$style_handles = array_merge(
-			$style_handles,
-			$block_type->style_handles,
-			$block_type->editor_style_handles
-		);
-
-		$script_handles = array_merge(
-			$script_handles,
-			$block_type->script_handles
-		);
+		// In older WordPress versions, like 6.0, these properties are not defined.
+		if ( isset( $block_type->style_handles ) && is_array( $block_type->style_handles ) ) {
+			array_merge( $style_handles, $block_type->style_handles );
```

Comment:
Will `array_merge` modify `$style_handles` here? Or should we assign the return value back to it?

Reply:
```php
$style_handles = array_merge( $style_handles, $block_type->style_handles );
```